module-4/backend/week_one.md

```diff
@@ -5,17 +5,30 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's the most useful thing you learned from completing the intermission week work?
+    * I think learning how files communicate with each other was very helpful. (Having to import and export files)
 2. What are some tools to help debug JavaScript code?
+    * Console.log, debugger, devtools
 3. What are some tools you need in order to unit test your JavaScript?
+    * Mocha with Chai.
 4. What is the syntax for invoking a function?
+    * function();
 5. What is CORS?
+    * Cross Origin Resource Sharing
 6. How do we enable CORS?
+    * 1. Open IIS manager, right click the site and go to properties, change to http headers tab, click add in custom http headers section and click add, enter access-control-allow-origin as the header name, enter * as the header value, click OK twice.
 7. What's `this` in JavaScript?
+    * THis is an object that references the current context of where it is invoked.
 8. What is Webpack and why is it useful?
+    * Webpack is way to organize assets and minify them so a browser can get the information that it needs. 
 9. When/why do you want to use event delegation?
+    * Setting an event listener on a parent object so as to not set one for every child element
```

Comment:
Especially for the cases where child elements have not yet been added to the DOM (and therefore couldn't have their own event listeners bound to them)